src/openrct2/peep/GuestPathfinding.cpp

```diff
@@ -1882,7 +1882,7 @@ sint32 guest_path_finding(rct_peep * peep)
         return guest_surface_path_finding(peep);
     }
 
-    TileCoordsXYZ loc = { peep->next_x / 32, peep->next_y / 32, peep->next_z / 8};
+    TileCoordsXYZ loc = { peep->next_x / 32, peep->next_y / 32, peep->next_z};
```

Comment:
Right, so `peep->next_x` and `..._y` use Big Coordinates, while `next_z` uses Tile Coordinates?
I think we should address that - in another PR, obviously.

Reply:
Annoyingly I think it will have to wait till new save format unless we want to do a conversion at import / export.